catroid/src/main/java/org/catrobat/catroid/io/StorageHandler.java

```diff
@@ -259,6 +259,8 @@
 	private static final StorageHandler INSTANCE;
 	private static final String TAG = StorageHandler.class.getSimpleName();
 	private static final String XML_HEADER = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\" ?>\n";
+	private static final String COPY_FILENAME_POSTFIX = "_copy";
```

Comment:
thought of that too, but to get non system strring resources, you need some sort of Context or Activity :( I think we should keep those out of here, maybe put it into the Constants class